Structure_Engine/Compute/Integrate.cs

```diff
@@ -95,6 +105,11 @@ public static partial class Compute
 
         /***************************************************/
 
+        [Description("Calculates all non-torsional section constants for a section profile based its edge curves and adjusts the profile curves to make them centred around the origin.")]
```

Comment:
Calculates all non-torsional section constants for a section profile based on its edge curves and translates the profile curves to be centred around the origin.